resources/website_sharing_whole_class_mosaic_plot_translation/MusicIcecream.csv

```diff
@@ -0,0 +1,9 @@
+"Age","Favorite","Music","Freq"
+"old","bubble gum","classical",1
+"old","bubble gum","rock",1
+"old","coffee","classical",3
+"old","coffee","rock",1
+"young","bubble gum","classical",2
+"young","bubble gum","rock",5
+"young","coffee","classical",1
+"young","coffee","rock",0
```

Comment:
There should be only ONE .Rmd file here. This should be deleted.

Reply:
The supporting documents can be put in the resources folder, it's ok to put it here.